### ISSUE TITLE
Feat: Include block sideband data in websocket messages

### ISSUE THREAD

**Issue by mistakia** [nanocurrency/nano-node#3322]
### Summary

Include block sideband data (`local_timestamp` and `height`) in websocket messages for related topics like `confirmations` and `new_unconfirmed_block`.

### What problem would be solved by this feature?

Would save an RPC request.

### Are there any previous requests for this feature?

None that I could find

### Do you have a suggested solution?

Provide it as a potential option on subscription.

### If this feature is approved, would you be willing to submit a pull request with the solution?

Not able to.

**Comment by gr0vity-dev** [comment]
Create a commit on my forked nano-node which adds these features :
https://github.com/gr0vity-dev/nano-node/commit/81ef11956c65bceb35219f10307f3a6b3f219cee
Also added the `hash` for `new_unconfirmed_block` because I think it's useful.

However I got no clue (yet) how to do a pull request. And not sure (yet) how to run the TEST cases.
New to this git stuff... Will figure it out eventually.

**Comment by dsiganos** [comment]
To run the test cases, if you are using my nano-workspace, which I think you are, do the following:
```
make
cd build
./core_test
```
You can also run specific test cases:
```
./core_test --gtest_filter=websocket.confirmation_options_sideband
./core_test --gtest_filter=websocket.*
```

You can make it repeat:
```
./core_test --gtest_filter=websocket.* --gtest_repeat=10
```

**Comment by dsiganos** [comment]
To create a pull request:
* go to https://github.com/nanocurrency/nano-node/pulls
* click "New Pull Request"
* click "compare across forks"
* in head repository, sleect your forked repository and branch

**Comment by zhyatt** [comment]
@gr0vity-dev I see you added a commit to your fork for this but didn't submit a PR. Can you open that PR with description `Closes #3322` so we can review?

**Comment by zhyatt** [comment]
Given the non-response by @gr0vity-dev we will be pulling the code from https://github.com/gr0vity-dev/nano-node/commit/6cfe054c1b368fc84828972bfeae0994a0029c41 and creating a PR, with @gr0vity-dev as co-author.
